pkg/alertmanager/multitenant_test.go

```diff
@@ -2462,11 +2462,12 @@ func TestComputeConfig(t *testing.T) {
 	mimirExternalURL := am.cfg.ExternalURL.String()
 
 	tests := []struct {
-		name   string
-		cfg    alertspb.AlertConfigDescs
-		expErr string
-		expCfg alertspb.AlertConfigDesc
-		expURL string
+		name       string
+		cfg        alertspb.AlertConfigDescs
+		expErr     string
+		expCfg     alertspb.AlertConfigDesc
+		expURL     string
+		expHeaders map[string]string
```

Comment:
This is the only new field in this test. I'm ensuring that we add the expected headers to the `amConfig` in each case.